src/plugins/hetero/tests/functional/hetero_synthetic.cpp

```diff
@@ -0,0 +1,46 @@
+// Copyright (C) 2018-2024 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "behavior/ov_plugin/hetero_synthetic.hpp"
+
+#include <vector>
+
+#include "openvino/core/visibility.hpp"
+
+#if defined(OPENVINO_ARCH_ARM) || defined(OPENVINO_ARCH_ARM64)
+const char* cpu_plugin_file_name = "openvino_arm_cpu_plugin";
+#elif defined(OPENVINO_ARCH_X86) || defined(OPENVINO_ARCH_X86_64)
+const char* cpu_plugin_file_name = "openvino_intel_cpu_plugin";
+#elif defined(OPENVINO_ARCH_RISCV64)
+const char* cpu_plugin_file_name = "openvino_riscv_cpu_plugin";
+#else
+#    error "Undefined system processor"
+#endif
+
+namespace {
+using ov::test::behavior::OVHeteroSyntheticTest;
+using ov::test::behavior::PluginParameter;
+
+// this tests load plugin by library name: this is not available during static linkage
+#ifndef OPENVINO_STATIC_LIBRARY
+
+// INSTANTIATE_TEST_SUITE_P(nightly_SingleMajorNode,
```

Comment:
Should be uncommented as it is inside `#ifdef`?

Reply:
@praasz  this is a test for CI validation, because the original PR always failed with large size error.

Reply:
Maybe remove it then ?